geospaas_harvesting/harvest.yml

```diff
@@ -3,43 +3,61 @@
 #  dump_on_interruption: False
 #  poll_interval: 600
 #  harvesters:
-#    OSISAF:
-#      class: 'OSISAFHarvester'
-#      max_fetcher_threads: 30
-#      # We exclude "EASE-Grid map projections" and "southern hemispheres" from harvesting process
-#      excludes: ['ease', '_sh_polstere',]
-#      max_db_threads: 1
-#      urls:
-#        - 'https://thredds.met.no/thredds/catalog/osisaf/met.no/ice/amsr2_conc/catalog.html'
-#        - 'https://thredds.met.no/thredds/catalog/osisaf/met.no/ice/conc/catalog.html'
-#    podaac:
-#      class: 'PODAACHarvester'
-#      max_fetcher_threads: 30
-#      max_db_threads: 1
-#      urls:
-#        - 'https://opendap.jpl.nasa.gov/opendap/allData/ghrsst/data/GDS2/L2P/VIIRS_NPP/NAVO/v1/2014/005/contents.html'
-#        - 'https://opendap.jpl.nasa.gov/opendap/allData/ghrsst/data/GDS2/L2P/VIIRS_N20/'
-#        - 'https://opendap.jpl.nasa.gov/opendap/allData/ghrsst/data/GDS2/L2P/VIIRS_NPP/'
-#        - 'https://opendap.jpl.nasa.gov/opendap/allData/ghrsst/data/GDS2/L2P/MODIS_A/'
-#    copernicus_sentinel:
-#      class: 'CopernicusSentinelHarvester'
-#      max_fetcher_threads: 30
-#      max_db_threads: 1
-#      url: 'https://scihub.copernicus.eu/apihub/search'
-#      search_terms:
-#        - 'platformname:Sentinel-1 AND NOT L0'
-#        - 'platformname:Sentinel-2 AND NOT L0'
-#        - 'platformname:Sentinel-3 AND NOT L0'
-#      username: 'username'
-#      # Environment variable name
-#      password: !ENV 'COPERNICUS_OPEN_HUB_PASSWORD'
-#    FTP_jaxa:
-#      class: 'FTPHarvester'
-#      max_fetcher_threads: 30
-#      max_db_threads: 1
-#      username: username
-#      password: !ENV 'JAXA_PASSWORD'
-#      fileformat: '.h5'
-#      urls:
-#        - 'ftp://ftp.gportal.jaxa.jp/standard/GCOM-W/GCOM-W.AMSR2/L3.SST_25/3/2012/07/'
+#   radarsat_local:
+#     class: 'LOCALHarvester'
+#     includes: ['RS2_.*[^\.xml][^\.kml][^\.xsd][^\.txt][^|\.DS_Store][^schemas]$']
```

Comment:
Two things:
- This argument should not be a list.
- This regex does not do what you think it does. An expression within square brackets contains a list of characters.
  For example, `[^\.xml]` matches one character which is not one of these characters: dot, "x", "m" or "l".
  If you want to match everything but a word, you need to use the `(?!...)` syntax. See https://docs.python.org/3/library/re.html#regular-expression-syntax.

Reply:
Done in the next commit